src/libraries/Common/src/Interop/Windows/BCrypt/Interop.BCryptSetProperty.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class BCrypt
+    {
+        [LibraryImport(Libraries.BCrypt, StringMarshalling = StringMarshalling.Utf16)]
+        internal static unsafe partial NTSTATUS BCryptSetProperty(
+            SafeBCryptHandle hObject,
+            string pszProperty,
+            void* pbInput,
+            int cbInput,
+            int dwFlags);
+
+        internal static unsafe void BCryptSetSZProperty(SafeBCryptHandle hObject, string pszProperty, string pszValue)
```

Comment:
I'm fine with the helper, I think, but once we're adding it we should use it.  So as a followup to this PR we should delete the `public static partial NTSTATUS BCryptSetProperty(SafeAlgorithmHandle hObject, string pszProperty, string pbInput, int cbInput, int dwFlags);` import in Interop\Windows\BCrypt\Cng.cs, and change all the callers of that to use this.